x-pack/plugins/siem/public/components/formatted_bytes/index.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as React from 'react';
+import { useContext } from 'react';
+import numeral from '@elastic/numeral';
+
+import {
+  AppKibanaFrameworkAdapter,
+  KibanaConfigContext,
+} from '../../lib/adapters/framework/kibana_framework_adapter';
+
+export const PreferenceFormattedBytes = React.memo<{ value: string | number }>(({ value }) => {
+  const config: Partial<AppKibanaFrameworkAdapter> = useContext(KibanaConfigContext);
+  return (
+    <>{config && config.bytesFormat ? numeral(value).format(config.bytesFormat) : `${value}b`}</>
```

Comment:
question - I am wondering if we should format the byte value anyway with the kibana default bytes (like hardcoded) just to make it cleaner.

Reply:
Good call, it definitely make sense to provide the best UX we can for the fallback/default scenario. It's what we were doing originally anyway, ha :)